plugins/jetty/pom.xml

```diff
@@ -0,0 +1,34 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.navercorp.pinpoint</groupId>
+        <artifactId>pom</artifactId>
+        <relativePath>../..</relativePath>
+        <version>1.5.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>pinpoint-jetty-plugin</artifactId>
+    <name>pinpoint-jetty-plugin</name>
+    <packaging>jar</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.navercorp.pinpoint</groupId>
+            <artifactId>pinpoint-profiler</artifactId>
```

Comment:
At runtime, pinpoint-profiler is not visible to Pinpoint profiler plugins. You should remove this dependency. If doing so makes this plugin build fail, maybe because the plugin uses types from pinpoint-profiler, then you must not use that type in the plugin.